src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -2201,6 +2202,11 @@ public function getBinaryStream($path)
             array($nodeId, $propertyName, $this->workspaceName)
         );
 
+        if (count($data) === 0) {
+            throw new RepositoryException('No binary data found in stream');
```

Comment:
it should be possible like this: https://github.com/phpcr/phpcr-api-tests/blob/master/fixtures/general/base.xml#L73-L77 . in the case of empty binary, it would look like `<sv:property sv:name="multivalue" sv:type="Binary" sv:multiple="true"/>` (and then hope that the fixtures loading does not choke on that :-) )